templates/bundle/resources/usr/local/bin/configure_liferay.sh

```diff
@@ -4,6 +4,13 @@ source /usr/local/bin/_liferay_bundle_common.sh
 source /usr/local/bin/_liferay_common.sh
 
 function main {
+	if [[ "${JAVA_VERSION}" == *21* ]] && [ ! -f "/opt/liferay/data/.inites7" ]
```

Comment:
@davisaints, can you be a little more specific when checking the Java version? Something like https://github.com/liferay/liferay-docker/commit/28765fc27c2b9f3f3f6376dccb1e7eef1ff48d4b.

My concern is we have in the future some patch release of another Java version that can incorrectly match this statement. For example: `JDK 1.8.21`. Does it make sense to you?

In addition, I'm checking the suggested solution to create an empty file and, to be honest, I'm concerned about its robustness. Can we discuss about it offline?